### PR TITLE
Projections

### PR DESCRIPTION
**Pull request by tischi**

### CODE REVIEW

**Comment by manerotoni** [comment]
Hi Tischi, 
I am going through the module. I wonder if we should mention that you can also project in time, for instance in the motivation. For showing dynamic processes this is quite useful.

**Comment by manerotoni** [comment]
On my OSX with recent Fiji, CLIJ2 projection does not conserve any properties. It gets back to pixels units. How did it worked for you?

**Comment by manerotoni** [comment]
@tischi 
You could merge now. I think with the current exercise it is fine.

**Review by manerotoni — COMMENTED** [review]
See my additional changes

File: _includes/projections/activities/projections_imagejgui_clij2.md [hunk 1]
```diff
@@ -9,32 +9,37 @@ Thus let's install the the very useful [update site](https://imagej.net/update-s
...
 
 ### Appearance of anisotropic images
-Notice that the images do not look correct in a physical sense, but squashed.
+Notice that the x and y projected images do not look correct in a physical sense, but squashed.
 This is due to a mismatch of the data (voxel) space and physical space.
 
 - Select one of the projected (x or y axis) images
 - `run("Properties...")`
-- Observe that the voxel sizes are correct, but the ImageJ viewer does not take them into account for rendering.
+- Observe that the voxel metadata is not maintained by CLIJ2
```
Comment (tischi) [inline _includes/projections/activities/projections_imagejgui_clij2.md:34]: @manerotoni very well spotted! this is an issue...
https://github.com/clij/clij2/issues/50

Comment (haesleinhuepf) [reply _includes/projections/activities/projections_imagejgui_clij2.md:34]: See also: https://clij.github.io/clij2-docs/faq#physical_units
:-)

Comment (manerotoni) [reply _includes/projections/activities/projections_imagejgui_clij2.md:34]: Mh @tischi,
have you commented on the latest version. I had already changed it. 


    Select one of the projected (x or y axis) images
    run("Properties...")
    Observe that the voxel metadata is not maintained by CLIJ2
    For y projection change the image properties to width = 0.0941345 and height = 0.4
    For x projection change the image properties to width = 0.4 and height = 0.0941345
    Observe that even with correct voxel size the ImageJ viewer does not take them into account for rendering.
    Opening the same image in BigDataViewer paints a different picture: [ Plugins > BigDataViewer > Open Current Image...]

Comment (manerotoni) [reply _includes/projections/activities/projections_imagejgui_clij2.md:34]: The latest version is in fact the correct one now. Forget my previous comment.

**Pull request merged**